resources/lib/objects/music.py

```diff
@@ -269,15 +258,13 @@ def song(self, item, e_item):
         if obj['Disc'] != 1:
             obj['Index'] = obj['Disc'] * 2 ** 16 + obj['Index']
 
-
         if update:
             self.song_update(obj)
         else:
             self.song_add(obj)
 
-
         self.link_song_album(*values(obj, QU.update_song_album_obj))
-        self.add_role(*values(obj, QU.update_role_obj)) # defaultt role
+        self.add_role(*values(obj, QU.update_role_obj))  # defaultt role
```

Comment:
Nitpicky spelling error that I'm betting we inherited

Reply:
In my defence, I was in full flake8 syntax mode when speeding past this line, all human text processing was disabled at the time :stuck_out_tongue_winking_eye: